scm_integrations.go

```diff
@@ -39,16 +40,24 @@ type SCMIntegrationsService interface {
 
 var _ SCMIntegrationsService = (*Client)(nil)
 
+type SCMIntegrationOwner struct {
+	ID       string `json:"id"`
+	Email    string `json:"email"`
+	Username string `json:"username"`
+}
```

Comment:
There is a `Owner` field in the `App` structure. It is currently embedded in this structure, but you may want to move it out of the `App` structure so that both `SCMIntegration` and `App` use the same `Owner` structure.